Python/Product/PythonTools/PythonTools/Navigation/EditFilter.cs

```diff
@@ -658,7 +658,10 @@ class NodeEnumerator<T> : IVsEnumNavInfoNodes where T : IVsNavInfoNode {
                             var line = pyPoint.Value.GetContainingLine();
                             var lineText = line.GetText();
```

Comment:
What does this do?  Can we get a comment?  It's a little disturbing that we're looking for a comment charater w/o using the tokenizer as it could be embedded in a string literal